spring-boot-starter/example-twitter/README.md

```diff
@@ -40,7 +40,7 @@ public class TwitterServletProcessApplication {
 
 3. You can also put BPMN, CMMN and DMN files in your classpath, they will be automatically deployed and registered within the process application.
 
-4. You can configure your Spring Boot application using `application.yaml` file. All possible Camunda-specific configuration parameters are listed [here](https://docs.camunda.org/manual/7.13/user-guide/spring-boot-integration/configuration/)
+4. You can configure your Spring Boot application using `application.yaml` file. All possible Camunda-specific configuration parameters are listed [here](https://docs.camunda.org/manual/7.14/user-guide/spring-boot-integration/configuration/)
```

Comment:
Should we simply go for `latest` here so it doesn't have to be upgraded every time?